src/main/java/com/reckue/post/models/Post.java

```diff
@@ -29,6 +29,7 @@
     private String source;
     private String username;
     private List<Tag> tags;
```

Comment:
Due to the fact that you had added new field in class Post, you also need to add such field in builder of method update in class PostServiceRealization, in class PostConverter, and may be in class PostControllerInt Test. But i'm not sure about the last.